homeassistant/components/tado/entity.py

```diff
@@ -34,7 +36,7 @@ class TadoHomeEntity(Entity):
     _attr_should_poll = False
     _attr_has_entity_name = True
 
-    def __init__(self, tado):
+    def __init__(self, tado: Any) -> None:
```

Comment:
Can we really not specify the type of `tado`?

Reply:
```suggestion
    def __init__(self, tado: TadoConnector) -> None:
```
From what I can see, it should be this (of course it needs a `from . import TadoConnector` at the top of the file)

Reply:
Indeed, this is more strict. Thanks! :)